sass/main.scss

```diff
@@ -782,10 +782,8 @@ footer {
     }
 }
 
-
 // Media queries for mobile
 
-
```

Comment:
Let's keep these newlines, helps keep the mobile section separate. Unless you think it's still clear without the extra space?

Reply:
[should be fixed here](https://github.com/codeRIT/brickhack.io/pull/1053/commits/aeddb50e352786a0ab75c83515170efe495ad791)